go/mysqlconn/client.go

```diff
@@ -363,13 +363,16 @@ func (c *Conn) parseInitialHandshakePacket(data []byte) (uint32, []byte, error)
 }
 
 func (c *Conn) writeHandshakeResponse41(capabilities uint32, cipher []byte, characterSet uint8, params *sqldb.ConnParams) error {
+	// Build our flags. If the server supported
```

Comment:
This new comment should be above the last line, which you added, to be closer to the code?